ssas/systems.go

```diff
@@ -135,6 +135,28 @@ func (system *System) GetClientTokens(trackingID string) ([]ClientToken, error)
 	return tokens, nil
 }
 
+func (system *System) DeleteClientToken(tokenID string) error {
+	db := GetGORMDbConnection()
+	defer Close(db)
+
+	tx := db.Begin()
+	defer func() {
+		if r := recover(); r != nil {
+			tx.Rollback()
+		}
+	}()
```

Comment:
When there's just one SQL statement, do we gain anything by using a transaction?

Reply:
@dhgreene it's two sql statements though

Reply:
But it doesn't have to be 😈

Reply:
On third reading, I see that you're wanting to delete both the root key and the client token. 👍